### PR TITLE
Chaos Labs - Risk and Simulation Platform for Aave v3 Proposal

### PR DESCRIPTION
**Pull request by Arieg419**

### CODE REVIEW

File: content/aips/8f46ded2d187-AIP-CHAOS-LABS-RISK-PLATFORM-FOR-V3.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+| Title                             | Status | Author        | Short Description                                      | discussions                                                                                                                                                                          | created     |
```
Comment (miguelmtzinf) [inline content/aips/8f46ded2d187-AIP-CHAOS-LABS-RISK-PLATFORM-FOR-V3.md:1]: Could you please use the structure that is detailed here?
https://github.com/aave/aip/blob/master/X-AIP.md?plain=1

**Review by miguelmtzinf — CHANGES_REQUESTED** [review]
How did you generate the filename hash?

**Comment by Arieg419** [comment]
I ran the following command to generate the filename hash:

`npm run generate-filename 'Chaos Labs Risk Platform Proposal' '2022-10-20'`

**Comment by miguelmtzinf** [comment]
> I ran the following command to generate the filename hash:
> 
> `npm run generate-filename 'Chaos Labs Risk Platform Proposal' '2022-10-20'`

The created date is '2022-10-19'. Could you please update the AIP text to reflect the date you used?

**Review by miguelmtzinf — APPROVED** [review]

**Pull request merged**